tools/windows/DatadogAgentInstaller/WixSetup/Datadog/AgentInstaller.cs

```diff
@@ -66,6 +66,7 @@ public Project ConfigureProject()
                 _agentCustomActions.ProcessDdAgentUserCredentials,
                 _agentCustomActions.DecompressPythonDistributions,
                 _agentCustomActions.ConfigureUser,
+                new Property("MsiLogging", "iwearucmop!"),
```

Comment:
If the log file is always created, where it will be created and is it overwritten by explicit params?

Reply:
I understood intention. Was not sure where it would be created. And what about second part of the question about explicit log parameters presence - would they overwrite implicit ones?